content/docs/rfcs/11/README.md

```diff
@@ -9,45 +9,63 @@ contributors:
   - Sanaz Taheri <sanaz@status.im>
 ---
 
-`WakuRelay` is part of the gossip domain for Waku. It is a thin layer on top of GossipSub.
+<!-- A note to contributors:
```

Comment:
I'd suggest to point to a common CONTRIBUTING.md file instead so that we don't need to change all RFCs to update guidelines.

Reply:
Thanks. I've removed this comment in aeae2b5. In fact, since there already exists [contributing guidelines](https://github.com/vacp2p/rfc#contributing) for this repo, it will be better to just add `sembr` to that.